curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/step-043.md

```diff
@@ -15,13 +15,13 @@ You should create a new `p` element below your element with the `Total Fat` text
 
 ```js
 assert(document.querySelector('.daily-value.sm-text')?.lastElementChild?.localName === 'p');
-assert(document.querySelector('.daily-value.sm-text')?.lastElementChild?.previousElementSibling?.innerText.match(/Total Fat 8g[\s|\n]+10%/));
+assert(document.querySelector('.daily-value.sm-text')?.lastElementChild?.previousElementSibling?.innerText.match(/Total Fat 8g[\s|\n]*10%/));
```

Comment:
```suggestion
assert(document.querySelector('.daily-value.sm-text')?.lastElementChild?.previousElementSibling?.innerText.match(/Total Fat 8g\s*10%/));
```

Reply:
Added as suggested and passing in safari and chrome.